roles/servicetelemetry/templates/manifest_snmp_traps.j2

```diff
@@ -0,0 +1,40 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: snmp-webhook
```

Comment:
For multiple STF deployments in the same namespace this will conflict. Also not sure the implications of multi-cloud in this configuration.

You should make use of the `{{ meta.name }}` and `{{ meta.namespace }}` parameters.

Reply:
this is still unsolved, to be done in next (follow-up patch)